src/sentry/web/frontend/react_page.py

```diff
@@ -10,8 +10,17 @@
 
 
 class ReactMixin:
-    def handle_react(self, request: Request) -> Response:
-        context = {"CSRF_COOKIE_NAME": settings.CSRF_COOKIE_NAME}
+    def meta_tags(self, request: Request, **kwargs):
+        return {}
+
+    def handle_react(self, request: Request, **kwargs) -> Response:
+        context = {
+            "CSRF_COOKIE_NAME": settings.CSRF_COOKIE_NAME,
+            "meta_tags": [
```

Comment:
Are websites that read these pages not smart enough to render the javascript to get the tags?

When we're splitting out sentry into a more pure SPA it's going to be harder to have django render views. So it would be nice if we _didn't_ add this functionality in django, and instead did it in the react view itself.

Reply:
Meta tags are generally handled by a SSR middleware, for example in Next.js this is what you would do: https://nextjs.org/docs/api-reference/next/head

And in Nuxt.js:
https://v3.nuxtjs.org/guide/features/head-management

There's no guarantee that these website will actually run the JavaScript. Some implementations might do it, others might not.

That facility probably needs to be built when we do the SPA conversion. In the mean time I think it's OK to do it this way as a growth project to measure the impact.

Reply:
👍 sounds good